writing-docs/author/index.rst

```diff
@@ -103,6 +103,15 @@ As you work, you can freely create new pages and directories and they will autom
 
 Content mapping is determined by :ref:`content mapping configuration files <control-map>` within the control repository. Open an issue on the control repository to discuss the addition of new content, or modify the content mapping files yourself in a pull request if you're also a site coordinator.
 
+.. _pull-request-builds:
+
+Previewing Changes
+------------------
+
+As you work, you'll want to see how the content you're writing will look rendered in context with the entire site. If your content repository is using a :term:`Strider` build, each time you *open a new pull request* or *push new commits to an existing pull request*, Strider will build a preview of your work to a staging environment. Once the build is complete, a bot account will post a comment on your pull request including a link to your personal preview.
+
```

Comment:
Consider:  Delete first sentence.

If your content repository is using a .... pull request (PR), Strider builds a preview of your work in a staging environment. After the build completes, a bot account posts a comment on your PR that includes a link to the staged content. Use the link to preview your work and share it with others.  

Question:  Will you have to provide access to your fork to enable others to view the staged content, or is it publicly accessible to all?  (One thing that I'm not clear about from the technical perspective -- In an environment not powered by an API/content store model, the content is built and then deployed to the server.  In the deconst world, the content goes to a content store, and is served via the API.  How does the built content get into the content store.  Is this part of the Strider build, or a separate process?  Maybe I should read the documentation.... :-).  The reason I bring it up is that I wonder whether it's helpful to know about the content store piece.  For example, if I get the link, and then go to it and see a 404 -- what would cause that problem-- the Strider build, or something else? )